test/UnionTest.cpp

```diff
@@ -554,7 +557,10 @@ TEST(Union, createSortedVariantWorksProperly) {
   }
   {
     qec->getQueryTreeCache().clearAll();
-    auto variant = unionOperation.createSortedVariant({0, 3, 1, 2});
+    auto action = unionOperation.createSortedClone({0, 3, 1, 2});
+    auto tree = std::move(action).getTree();
```

Comment:
no need for the `action` variable:
`auto tree = union.createSorted().getTree()`.